Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -1777,6 +2124,594 @@ RefPtr<StyleValue> Parser::parse_box_shadow_value(ParsingContext const& context,
     return BoxShadowStyleValue::create(offset_x, offset_y, blur_radius, color);
 }
 
+RefPtr<StyleValue> Parser::parse_flex_value(ParsingContext const& context, Vector<StyleComponentValueRule> const& component_values)
+{
+    auto is_flex_grow_or_shrink = [](StyleValue const& value) -> bool {
+        if (value.is_numeric())
+            return true;
+        return false;
+    };
+
+    auto is_flex_basis = [](StyleValue const& value) -> bool {
+        if (value.is_length())
+            return true;
+        if (value.to_identifier() == ValueID::Content)
+            return true;
+        return false;
+    };
+
+    if (component_values.size() == 1) {
+        auto value = parse_css_value(context, PropertyID::Flex, component_values[0]);
+        if (!value)
+            return nullptr;
+
+        auto zero = NumericStyleValue::create(0);
+        auto one = NumericStyleValue::create(1);
+        auto auto_ident = IdentifierStyleValue::create(ValueID::Auto);
```

Comment:
There are paths which don't need all of those. It's a bit sad to let the unused Values starve.

Reply:
Sure. I realised while looking at this too, that the whole `is_initial()` case is unnecessary, if I update the values in Properties.json to have the correct initial values.